bootstrap-salt.sh

```diff
@@ -2542,7 +2542,7 @@ __install_pip_pkgs() {
 #----------------------------------------------------------------------------------------------------------------------
 __install_tornado_pip() {
     # OS needs tornado <5.0 from pip
-    __check_pip_allowed "You need to allow pip based installations (-P) for Tornado <5.0 in order to install Salt on Python 3"
+    __check_pip_allowed "You need to allow pip based installations \(-P\) for Tornado <5.0 in order to install Salt on Python 3"
```

Comment:
The `()` were not being shown on the output?

Reply:
bash -n was complaining about the un-escaped ()

Reply:
Hmm, I really don't want to see back slashes on the output...
Can you confirm the echo's don't include the `\`'s please.

Reply:
will remove them

Reply:
done